protobuf/v1/snapshot.proto

```diff
@@ -13,6 +13,8 @@ service SnapshotRpc {
     rpc CreateNexusSnapshot(NexusCreateSnapshotRequest) returns (NexusCreateSnapshotResponse) {}
     rpc ListSnapshot(ListSnapshotsRequest) returns (ListSnapshotsResponse) {}
     rpc DestroySnapshot(DestroySnapshotRequest) returns (google.protobuf.Empty) {}
+    rpc CreateSnapshotClone(CreateSnapshotCloneRequest) returns (Replica) {}
```

Comment:
Wondering if should name these as `CreateCloneVolume` , `CreateCloneVolumeRequest` ?

Reply:
There's no volume at this level, only nexuses, replicas and snapshots.
IMHO what we have here is appropriate as it is specifically for snapshot cloning.